examples/shaders/shaders_basic_pbr.c

```diff
@@ -105,11 +106,11 @@ int main()
     shader.locs[SHADER_LOC_MAP_NORMAL] = GetShaderLocation(shader, "normalMap");
     // WARNING: Similar to the MRA map, the emissive map packs different information 
     // into a single texture: it stores height and emission data
-    // It is binded to SHADER_LOC_MAP_EMISSION location an properly processed on shader
+    // It is binded to the SHADER_LOC_MAP_EMISSION location and properly processed on the shader
     shader.locs[SHADER_LOC_MAP_EMISSION] = GetShaderLocation(shader, "emissiveMap");
     shader.locs[SHADER_LOC_COLOR_DIFFUSE] = GetShaderLocation(shader, "albedoColor");
 
-    // Setup additional required shader locations, including lights data
+    // Setup additional required shader locations, including the lights data
```

Comment:
I'm afraid I'm not using `the` in any of the shaders examples, for consitency, I prefer to keep it without it.